lib/s3_meta_sync/syncer.rb

```diff
@@ -68,7 +71,8 @@ def download(source, destination)
       log "Downloading: #{download.size} Deleting: #{delete.size}", true
 
       if download.any? || delete.any?
-        Dir.mktmpdir do |staging_area|
+        Dir.mktmpdir(TEMP_FOLDER_PREFIX) do |staging_area|
+          log "Temp folder: #{staging_area}"
```

Comment:
maybe call it `Staging area` here too since that is what we name it internally ... and hopefully in the docs too ...